ui/raidboss/data/05-shb/raid/e4s.js

```diff
@@ -458,63 +374,20 @@
       regexJa: Regexes.startsUsing({ id: '4135', source: 'マキシタイタン', capture: false }),
       regexCn: Regexes.startsUsing({ id: '4135', source: '极大泰坦', capture: false }),
       regexKo: Regexes.startsUsing({ id: '4135', source: '거대 타이탄', capture: false }),
-      infoText: {
-        en: 'Knockback',
-        de: 'Rückstoß',
-        fr: 'Poussée',
-        ja: 'ノックバック',
-        cn: '击退',
-        ko: '넉백',
-      },
+      response: Responses.knockback(),
     },
     {
       id: 'E4S Weight of the World',
       regex: Regexes.headMarker({ id: '00BB' }),
       condition: function(data, matches) {
         return data.me == matches.target;
       },
-      alertText: {
-        en: 'Weight, Get Out',
-        ja: '青: 離れて',
-        de: 'Schwere, Raus gehen',
-        fr: 'Poids, éloignez-vous',
-        cn: '蓝色：离开人群',
-        ko: '파랑징, 피하기',
-      },
+      response: Responses.getOut(),
     },
     {
       id: 'E4S Megalith',
       regex: Regexes.headMarker({ id: '005D' }),
-      alertText: function(data, matches) {
-        if (data.role != 'tank') {
-          return {
-            en: 'Away from Tanks',
-            ja: 'タンクから離れて',
-            de: 'Weg von den Tanks',
-            fr: 'Loin des tanks',
-            cn: '远离坦克',
-            ko: '탱커에서 멀어지기',
-          };
-        }
-        if (matches.target == data.me) {
-          return {
-            en: 'Stack on YOU',
-            ja: '自分にシェア',
-            de: 'Auf DIR sammeln',
-            fr: 'Package sur VOUS',
-            cn: '集合分摊',
-            ko: '쉐어징 대상자',
-          };
-        }
-        return {
-          en: 'Stack on ' + data.ShortName(matches.target),
-          ja: data.ShortName(matches.target) + 'にシェア',
-          de: 'Auf ' + data.ShortName(matches.target) + ' sammeln',
-          fr: 'Package sur ' + data.ShortName(matches.target),
-          cn: '集合 ->' + data.ShortName(matches.target),
-          ko: '"' + data.ShortName(matches.target) + '" 쉐어징',
-        };
-      },
+      response: Responses.stackOn(),
```

Comment:
This is not correct.  For non-tanks it is away from tanks.  For tanks it is stack.